user-interface/src/login/broadcast-logout.test.ts

```diff
@@ -10,7 +10,7 @@ describe('Broadcast Logout', () => {
     const postMessageSpy = vi
       .spyOn(BroadcastChannelHumble.prototype, 'postMessage')
       .mockReturnValue();
-    // eslint-disable-next-line @typescript-eslint/no-unsafe-function-type
+    // biome-ignore lint: do not use Function as a type
```

Comment:
This is the other thing that I don't really like, and actually fairly strongly. The fact that the ignore cannot target specific rules means that we could end up violating rules we actually care about because we are trying to intentionally ignore a different rule in just this one case.

Reply:
https://biomejs.dev/linter/#ignore-code